data/mock-28/sample-metadata.tsv

```diff
@@ -0,0 +1,3 @@
+SampleID	BarcodeSequence	LinkerPrimerSequence	ReversePrimer	PrimerName	Description
+Mock.nifH.even	NA			Ueda19F-R6	Mock.nifH.even
```

Comment:
The barcode sequence should correspond to the MiSeq run that contains these samples.

Also need to add forward and reverse primer sequences here.